src/main/java/igrad/logic/commands/requirement/RequirementAddCommand.java

```diff
@@ -15,17 +15,20 @@
 public class RequirementAddCommand extends RequirementCommand {
     public static final String COMMAND_WORD = REQUIREMENT_COMMAND_WORD + "add";
 
-    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Adds a requirement.\n"
-        + "Parameter: "
+    public static final String MESSAGE_DETAILS = COMMAND_WORD + ": Adds a requirement.\n";
+
+    public static final String MESSAGE_USAGE = "Parameter: "
         + PREFIX_NAME + "NAME "
-        + PREFIX_CREDITS + "CREDITS_NEEDED\n"
+        + PREFIX_CREDITS + "CREDITS_TO_FULFIL\n"
         + "Example: " + COMMAND_WORD + " "
         + PREFIX_NAME + "Unrestricted Electives "
         + PREFIX_CREDITS + "24\n";
 
+    public static final String MESSAGE_HELP = MESSAGE_DETAILS + MESSAGE_USAGE;
+
     public static final String MESSAGE_REQUIREMENT_ADD_SUCCESS = "New requirement added: %1$s";
     public static final String MESSAGE_REQUIREMENT_NOT_ADDED = "Added requirement must be provided with arguments "
-        + PREFIX_NAME + "TITLE " + PREFIX_CREDITS + "CREDITS ";
+        + PREFIX_NAME + "NAME " + PREFIX_CREDITS + "CREDITS ";
```

Comment:
ok thanks for changing  it back to `NAME`, sorry i didn't changed it properly the  last time